### DISCUSSION TITLE
Disable switching to Smartphone Interface on connection

### DISCUSSION THREAD

**Discussion by cichacz** [Mr-MIBonk/M.I.B._More-Incredible-Bash#173] [Ideas]
Hello,

first of all I’d like to thank you for this awesome project. Subwoofer knob works perfectly.

I have one idea that seems simple (maybe it’s not 😅). I got myself a Carlinkit for wireless CarPlay and the one thing that irritates me is that MMI switches to the ASI interface every time when initiating the connection. Sometimes it’s after few seconds while I’m in the middle of for example changing music.
I was wondering whether it’s possible to remove this behavior?
So the outcome should be that ASI gets initialized in the background and one can go there via menu whenever needed but no automatic “redirect” happens.

**Comment by harman-f** [comment]
Cane you share a video of this behaviour?

**Comment by cichacz** [comment]
https://youtu.be/CDSgYMx3rGw
In this video I was only pressing the “radio” button. All other changes happened automatically.
I think it would be great if MMI stayed on radio and ASI initialized in the background.
